.github/workflows/build.yml

```
@@ -21,3 +21,39 @@ jobs:
           java-version: '11'
       - run: ./gradlew clean
       - run: ./gradlew build
+
+  build-server-local:
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, windows-latest, macOS-latest ]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - run: |
+          cd ..
+          git clone https://github.com/JetBrains/projector-server.git
+          cd projector-demo
+          echo projectorServerSource=local > local.properties
+      - run: ./gradlew clean
+      - run: ./gradlew build
+
+  build-server-self-hosted-repos:
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, windows-latest, macOS-latest ]
+        projectorServerSource: [ jitpack-cb4aea89, space-cb4aea89 ]
```

Minor: why don't we use the latest version for JitPack here?

```suggestion
        projectorServerSource: [ jitpack--SNAPSHOT, space-cb4aea89 ]
```

---

My mistake. I copied this from the example. In the example, the specific version seemed to be more descriptive.